src/codelabs/async-await.md

```diff
@@ -231,58 +275,62 @@ your window is wide enough — is to the right of the synchronous example.
 <div class="col-sm" markdown="1">
 #### Example: synchronous functions
 
+<?code-excerpt "async_await/bin/get_order_sync_bad.dart (no-warning)" replace="/(\s+\/\/ )(Imagine.*? is )(.*)/$1$2$1$3/g"?>
 ```dart
-// Synchronous
 String createOrderMessage() {
   var order = getUserOrder();
   return 'Your order is: $order';
 }
 
-Future<String> getUserOrder() {
-  // Imagine that this function is
-  // more complex and slow.
-  return
+Future<String> getUserOrder() =>
+    // Imagine that this function is
+    // more complex and slow
     Future.delayed(
-      Duration(seconds: 4), () => 'Large Latte');
-}
+      Duration(seconds: 2),
+      () => 'Large Latte',
+    );
 
-// Synchronous
-main() {
+void main() {
   print('Fetching user order...');
   print(createOrderMessage());
 }
+```
 
-// 'Fetching user order...'
-// 'Your order is: Instance of _Future<String>'
+{:.console-output}
+```nocode
+Fetching user order...
+Your order is: Instance of _Future<String>
 ```
 </div>
 <div class="col-sm" markdown="1">
 #### Example: asynchronous functions
 
+<?code-excerpt "async_await/bin/get_order.dart" replace="/(\s+\/\/ )(Imagine.*? is )(.*)/$1$2$1$3/g; /async|await/[!$&!]/g; /(Future<\w+\W)( [^g])/[!$1!] $2/g; /4/2/g"?>
 {% prettify dart %}
-// Asynchronous
-[!Future<String>!] createOrderMessage() [!async!] {
-var order = [!await!] getUserOrder();
-return 'Your order is: $order';
+[!Future<String>!]  createOrderMessage() [!async!] {
+  var order = [!await!] getUserOrder();
+  return 'Your order is: $order';
 }
 
-Future<String> getUserOrder() {
-// Imagine that this function is
-// more complex and slow.
-return
-  Future.delayed(
-    Duration(seconds: 4), () => 'Large Latte');
-}
+Future<String> getUserOrder() =>
+    // Imagine that this function is
+    // more complex and slow
+    Future.delayed(
+      Duration(seconds: 2),
+      () => 'Large Latte',
+    );
 
-// Asynchronous
-main() [!async!] {
-print('Fetching user order...');
-print([!await!] createOrderMessage());
+[!Future<void>!]  main() [!async!] {
```

Comment:
This change needs to be reflected in the "three ways" list that follows... if we keep this change.

Reply:
Reverted.